Sources/OpenSSL/shim.h

```diff
@@ -196,4 +196,27 @@ static inline void EVP_CIPHER_CTX_free_wrapper(EVP_CIPHER_CTX *ctx) {
         #endif
 }
 
+// This wrapper allows for a common call for both versions of OpenSSL when setting other keys for RSA.
+static inline void RSA_set_keys(RSA *rsakey, BIGNUM *n, BIGNUM *e, BIGNUM *d, BIGNUM *p, BIGNUM *q, BIGNUM *dmp1, BIGNUM *dmq1, BIGNUM *iqmp) {
+
+	#if (OPENSSL_VERSION_NUMBER < 0x10100000L)
```

Comment:
Please use `OPENSSL_VERSION_NUMBER >= 0x10100000L` to match the rest of the file.

Reply:
Sure, Thanks @ianpartridge